docs/asset-store/docs/06-01-asset.md

```diff
@@ -88,13 +95,19 @@ This table lists all possible parameters of a given resource together with their
 | **spec.source.filter** |    **NO**   | Specifies the regex pattern used to select files to store from the package. |
 | **spec.source.validationwebhookservice** |    **NO**   | Provides specification of the validation webhook services. |
 | **spec.source.validationwebhookservice.name** |    **NO**   | Provides the name of the validation webhook service. |
-| **spec.source.validationwebhookservice.namespace** |    **NO**   | Provides the Namespace in which the service is available. It must be the same as the asset's Namespace. |
+| **spec.source.validationwebhookservice.namespace** |    **NO**   | Provides the Namespace in which the service is available. |
 | **spec.source.validationwebhookservice.endpoint** |    **NO**   | Specifies the endpoint to which the service sends calls. |
+| **spec.source.validationwebhookservice.filter** |    **NO**   | Specifies the regex pattern used to select files to send to the service. |
 | **spec.source.mutationwebhookservice** |    **NO**   | Provides specification of the mutation webhook services. |
 | **spec.source.mutationwebhookservice.name** |    **NO**   | Provides the name of the mutation webhook service. |
-| **spec.source.mutationwebhookservice.namespace** |    **NO**   | Provides the Namespace in which the service is available. It must be the same as the asset's Namespace. |
+| **spec.source.mutationwebhookservice.namespace** |    **NO**   | Provides the Namespace in which the service is available. |
 | **spec.source.mutationwebhookservice.endpoint** |    **NO**   | Specifies the endpoint to which the service sends calls. |
-| **spec.source.mutationwebhookservice.metadata** |    **NO**   | Provides detailed metadata specific for a given mutation service and its functionality. |
+| **spec.source.mutationwebhookservice.filter** |    **NO**   | Specifies the regex pattern used to select files to send to the service. |
+| **spec.source.metadatawebhookservice** |    **NO**   | Provides specification of the metadata webhook services. |
```

Comment:
```suggestion
| **spec.source.metadatawebhookservice** |    **NO**   | Provides specification of the metadata webhook service. |
```
Btw could you also correct for spec of the validation and mutation services? (lines 96 and 101)?

Reply:
But this variable provided array od services